src/server/saved_objects/client/lib/create_id_query.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Finds a document by either its v5 or v6 format
+ *
+ * @param type The documents type
+ * @param id The documents id or legacy id
+**/
+export function createIdQuery({ type, id }) {
+  return {
+    version: true,
+    size: 1,
+    query: {
+      bool: {
+        should: [
+          // v5 document
```

Comment:
When attempting to import that dashboard I get an error that the document already exists:

```
curl -X POST -H "Content-Type: application/json" -H "kbn-xsrf: 6.0.0-alpha3" -d '{ "version": "6.0.0-alpha3", "objects": [{ "id": "test", "type": "dashboard", "attributes": { "title": "My Test Dashboard From V6" }}]}' http://localhost:5601/api/kibana/dashboards/import
{"objects":[{"id":"test","type":"dashboard","attributes":{"title":"My Test Dashboard From V6"},"error":{"message":"[dashboard][test]: version conflict, document already exists (current version [3])"}}]}
```

Ping me tomorrow and let's go over this live.